diesel/Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 
 [dependencies]
 byteorder = "1.0"
-diesel_derives = "~1.4.0"
+diesel_derives = { path = "../diesel_derives", version = "~1.4.0" }
```

Comment:
Can we change this to 
```toml
[dependencies.diesel_derives]
version = "~1.4.0"
path = "../diesel_derives"
```
as this makes it easier to change the version?
(Same for all other changes)